Rename some files to reflect that this is Dante, not the original SOCKS:
- /etc/socks.conf => /etc/dante.conf
- /etc/sockd.conf => /etc/danted.conf
- /usr/sbin/sockd => /usr/sbin/danted

--- a/INSTALL
+++ b/INSTALL
@@ -264,8 +264,8 @@
 
 No configuration-files are installed, these must be installed
 manually.
-The Dante server expects it's configuration file to be in /etc/sockd.conf.
-The client library uses the file /etc/socks.conf.
+The Dante server expects it's configuration file to be in /etc/danted.conf.
+The client library uses the file /etc/dante.conf.
 
 The file bin/socksify.sh contains a shell script that allows one
 to dynamically socksify a dynamicly linked application at runtime.
--- a/doc/sockd.8
+++ b/doc/sockd.8
@@ -40,11 +40,11 @@
 .\" any improvements or extensions that they make and grant Inferno Nettverk A/S
 .\" the rights to redistribute these changes.
 .\"
-.TH SOCKD 8 "Feb 17, 2001"
+.TH DANTED 8 "Feb 17, 2001"
 .SH NAME
-sockd \- network proxyserver
+danted \- network proxyserver
 .SH SYNOPSIS
-.B sockd
+.B danted
 .RB [ \-DLVdhnv ]
 .RB [ \-N
 .IR number ]
@@ -112,7 +112,7 @@
 be set to a (local) filesystem with low latency.
 .SH FILES
 .TP
-/etc/sockd.conf
+/etc/danted.conf
 .B Dante
 server configuration file.
 .SH AUTHORS
@@ -120,7 +120,7 @@
  Michael Shuldman <michaels@inet.no>: Design and implementation.
  Karl-Andre' Skevik <karls@inet.no>: Autoconf and porting.
 .SH SEE ALSO
-socks.conf(5) sockd.conf(5)
+dante.conf(5) danted.conf(5)
 .PP
 Information about new releases and other related issues can be found
 on the
--- a/doc/sockd.conf.5
+++ b/doc/sockd.conf.5
@@ -40,9 +40,9 @@
 .\" any improvements or extensions that they make and grant Inferno Nettverk A/S
 .\" the rights to redistribute these changes.
 .\"
-.TH SOCKD.CONF 5 "May 11, 2001"
+.TH DANTED.CONF 5 "May 11, 2001"
 .SH NAME
-sockd.conf \- \fBDante\fP server configuration file syntax
+danted.conf \- \fBDante\fP server configuration file syntax
 .SH DESCRIPTION
 The configuration file for the \fBDante\fP server controls both access
 controls and logging.  It is divided into three parts; server settings,
@@ -308,7 +308,7 @@
 .SH ROUTES
 The routes are specified with a \fBroute\fP keyword.
 Inside a pair of parens ({}) a set of keywords control the behavior of
-the route.  See socks.conf(5) for a description.  This is used to
+the route.  See dante.conf(5) for a description.  This is used to
 perform so-called "server-chaining", where one socks-server connects
 to another socks-server futher upstream.  
 .SH EXAMPLES
@@ -316,7 +316,7 @@
 .SH FILES
 .nf
 .ta \w 1
-/etc/sockd.conf	\fBDante\fP server configuration file.
+/etc/danted.conf	\fBDante\fP server configuration file.
 /etc/passwd	file used when checking username/passwords.
 .fi
 .SH AUTHORS
@@ -324,7 +324,7 @@
  Michael Shuldman <michaels@inet.no>: Design and implementation.
  Karl-Andre' Skevik <karls@inet.no>: Autoconf and porting.
 .SH SEE ALSO
-sockd(8), socks.conf(5), hosts_access(5)
+danted(8), dante.conf(5), hosts_access(5)
 .Pp
 Information about new releases and other related issues can be found
 on the
--- a/doc/socks.conf.5
+++ b/doc/socks.conf.5
@@ -40,9 +40,9 @@
 .\" any improvements or extensions that they make and grant Inferno Nettverk A/S
 .\" the rights to redistribute these changes.
 .\"
-.TH SOCKS.CONF 5 "February 17, 2001"
+.TH DANTE.CONF 5 "February 17, 2001"
 .SH NAME
-socks.conf \- socks client configuration file syntax
+dante.conf \- socks client configuration file syntax
 .SH DESCRIPTION
 The configuration file for the socks client library allow control over
 logging and server selection.  It is divided into two parts,
@@ -119,13 +119,13 @@
 authentication.  Not recommended as other users on the system 
 might be able to see your password.
 .SH FILES
-.I /etc/socks.conf
+.I /etc/dante.conf
 .SH AUTHORS
 For Inferno Nettverk A/S, Norway:
  Michael Shuldman <michaels@inet.no>: Design and implementation.
  Karl-Andre' Skevik <karls@inet.no>: Autoconf and porting.
 .SH SEE ALSO
-sockd(8) sockd.conf(5)
+danted(8) danted.conf(5)
 .Pp
 Information about new releases and other related issues can be found
 on the
